### PR TITLE
fix: pin Windows images

### PR DESCRIPTION
**Pull request by rbtr**
**Reason for Change**:

Latest Windows LTSC2019 image is borked. https://github.com/microsoft/Windows-Containers/issues/493
Pins to previous monthly release, and pins the LTSC2022 image to current working one for good measure.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Review by jpayne3506 — APPROVED** [review]

File: npm/windows.Dockerfile [hunk 1]
```diff
@@ -7,7 +7,13 @@ WORKDIR /usr/local/src
 COPY . .
 RUN GOOS=windows CGO_ENABLED=0 go build -v -o /usr/local/bin/azure-npm.exe -ldflags "-X main.version="$VERSION" -X "$NPM_AI_PATH"="$NPM_AI_ID"" -gcflags="-dwarflocationlists=true" npm/cmd/*.go
 
-FROM mcr.microsoft.com/windows/servercore:${OS_VERSION}
+# intermediate for win-ltsc2019
+FROM mcr.microsoft.com/windows/servercore@sha256:6fdf140282a2f809dae9b13fe441635867f0a27c33a438771673b8da8f3348a4 as ltsc2019
```
Comment (jpayne3506) [inline npm/windows.Dockerfile:11]: NPM does not run on ltsc 2019. Can update this to just use 2022

**Review by jpayne3506 — CHANGES_REQUESTED** [review]

File: npm/windows.Dockerfile (hunk 1 above)
Comment (rbtr) [reply npm/windows.Dockerfile:11]: it will only build the image for 2019 if that's the only OS_VERSION the NPM image build target has

**Review by jpayne3506 — APPROVED** [review]

**Review by camrynl — APPROVED** [review]

**Review by huntergregory — APPROVED** [review]

**Review by jpayne3506 — APPROVED** [review]
🚀

**Review by camrynl — APPROVED** [review]

**Pull request merged**